modules/apps/remote-app/remote-app-service/src/main/java/com/liferay/remote/app/internal/upgrade/v2_0_0/RemoteAppEntryUpgradeProcess.java

```diff
@@ -49,6 +49,12 @@ protected void doUpgrade() throws Exception {
 			RemoteAppEntryTable.class,
 			new AlterColumnName("url", "iFrameURL STRING null"));
 
+		if (!hasColumn("RemoteAppEntry", "portletAlias")) {
+			alter(
+				RemoteAppEntryTable.class,
+				new AlterTableAddColumn("portletAlias", "VARCHAR(255)"));
```

Comment:
This should be `VARCHAR(75)` as shown [here](https://github.com/liferay-frontend/liferay-portal/pull/1488/commits/ffca2c7df617152c491f3958237458d4aecce37d#diff-2d210b2c1c29f229b4a75c68a24f1a629a60f843c030d92480d8790279073084R15)

Or otherwise, you must change `portlet-model-hints.xml` and rerun `gradle buildService` so that everything is aligned